### ISSUE TITLE
Login history

### ISSUE THREAD

**Issue by deepj** [eventil/eventil.com#80]
I'd appreciate to see my last (20?) activities including dates and times and IP locations

**Comment by zaiste** [comment]
@deepj can you elaborate and/or give an example ?

**Comment by deepj** [comment]
@zaiste Yes, take a look at https://twitter.com/settings/your_twitter_data and on **Login History**.
